src/components/Banner/Banner.css

```
@@ -0,0 +1,31 @@
+p, h1 {
+  margin: 0;
+}
+
+.banner-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin-top: 60px;
+}
+
+.about {
+  font-weight: 600;
+  color: #8E2424;
+  line-height: 24px;
+}
+
+.title {
+  font-size: 48px;
+  color: #101828;
+  line-height: 60px;
+  margin: 24px 0 36px 0;
```

o da vertical (top e bottom) se mantem `0` se não colocar o último

```suggestion
  margin: 24px 0 36px;
```